SU-HSDO/suhumsci
# Sparkbox Sprint 8

# READY FOR REVIEW

## Summary
- WYSIWYG Styles [STN-127](https://sparkbox.atlassian.net/browse/STN-127) | PR #494 
- Local Footer Styles and Implementation [STN-172](https://sparkbox.atlassian.net/browse/STN-172) | PR #496 
- QA Feedback from last sprint [STN-184](https://sparkbox.atlassian.net/browse/STN-184) | PR #497 
- Hero option 1 styles [STN-125](https://sparkbox.atlassian.net/browse/STN-125) | PR #495 

✅ Passing CodeClimate: https://codeclimate.com/github/SU-HSDO/suhumsci/pull/498

## Need Review By (Date)
asap (2/25)

## Urgency
high

## Steps to Test
- ⚠️ TODO: We've verified that this branch passes the code climate and circle CI tests against the develop branch.

1. See other PRs "steps to test" if you'd like to test locally for those changes.
2. Verify code looking at the file changes here.
3. Go into `humsci_basic` folder directory and run `npm i` and `npm run test` to verify tests pass.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
